Formula/gnupg.rb

```diff
@@ -34,32 +34,21 @@ class Gnupg < Formula
 
   uses_from_macos "sqlite", since: :catalina
 
-  on_linux do
-    depends_on "libidn"
-  end
```

Comment:
No idea where this is from. Didn't see dependency in Arch or Debian so trying without it.

Reply:
looks like the git blame does not work either 😆 
<img width="473" alt="image" src="https://user-images.githubusercontent.com/1580956/211183273-01d577dd-5c27-4e6b-8c3a-57c6071a978e.png">